src/scss/partials/footer.scss

```diff
@@ -1,114 +1,224 @@
 .footer {
-	position: relative;
-	padding: 20px 20px 5px;
-	border-top: 10px solid var.$blue;
-	text-align: center;
-
-	nav {
-		ul {
-			display: flex;
-			flex-wrap: wrap;
-			justify-content: center;
-		}
-
-		a {
-			padding: 10px;
-			text-decoration: none;
-
-			&:hover {
-				text-decoration: underline;
-			}
-		}
-	}
-
-	a {
-		color: inherit;
-		text-decoration: underline;
-
-		svg {
-			height: 3em;
-			fill: var.$blue;
-			color: var.$white;
-		}
-
-		&:hover {
-			text-decoration: none;
-
-			svg {
-				fill: var.$white;
-				color: var.$blue;
-			}
-		}
-	}
+  position: relative;
+  padding: 20px 20px 5px;
+  border-top: 10px solid var.$blue;
+
+  text-align: left;
+  font-family: Inter;
+  font-style: normal;
+  font-weight: normal;
```

Comment:
Try and use weight numbers rather than words, this would be `400`. Gives more control over it